### ISSUE TITLE
xgboost4j_2.13 version 2.0.2 pom.xml uses a property in the parent definition. This is prohibited by maven definition.

### ISSUE THREAD

**Issue by imotsch** [dmlc/xgboost#9898]
group ml.dmlc:xgboost-jvm_2.13:2.0.2
artifact ml.dmlc:xgboost4j_2.13:2.0.2

The pom.xml looks like this. Any chance to eliminate that `${scala.binary.version}` and write the explicit number `2.13` here?
We are using apache-maven-3.9.4
See also https://maven.apache.org/maven-ci-friendly.html
and the orgin of the message from latest intellij at https://youtrack.jetbrains.com/issue/IDEA-249179

```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>
    <parent>
        <groupId>ml.dmlc</groupId>
        <artifactId>xgboost-jvm_${scala.binary.version}</artifactId>
        <version>2.0.2</version>
    </parent>
    <name>xgboost4j</name>
    <artifactId>xgboost4j_${scala.binary.version}</artifactId>
    <version>2.0.2</version>
    <packaging>jar</packaging>
```

**Comment by hcho3** [comment]
We just released the 2.0.3 patch release with the correct metadata:
```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>ml.dmlc</groupId>
    <artifactId>xgboost-jvm_2.13</artifactId>
    <version>2.0.3</version>
    <packaging>pom</packaging>
...
```

**Comment by imotsch** [comment]
Thank you very much for that fast response. I appreciate your good services.
